apps/vault/contracts/Vault.sol

```diff
@@ -50,12 +58,12 @@ contract Vault is EtherTokenConstant, AragonApp, DepositableStorage {
         external
         authP(TRANSFER_ROLE, arr(_token, _to, _value))
     {
-        require(_value > 0);
+        require(_value > 0, ERROR_TRANSFER_VALUE_ZERO);
 
         if (_token == ETH) {
             _to.transfer(_value);
```

Comment:
Wondering if we should use a `require(_to.send(_value))` here so we can customize the error message.